packages/graphql-generator/package.json

```diff
@@ -17,18 +17,23 @@
     "aws"
   ],
   "scripts": {
-    "build": "tsc",
+    "build": "tsc && yarn check-codegen-version",
     "watch": "tsc -w",
     "test": "jest",
     "test-watch": "jest --watch",
-    "extract-api": "ts-node ../../scripts/extract-api.ts"
+    "extract-api": "ts-node ../../scripts/extract-api.ts",
+    "check-codegen-version": "bash ./scripts/check-codegen-version.sh"
```

Comment:
Adding a command that makes sure our vendor code matches our dependency during the build process